CMakeLists.txt

```
@@ -369,3 +369,28 @@ install(
   RUNTIME DESTINATION bin
   BUNDLE DESTINATION bin
   LIBRARY DESTINATION lib)
+
+# Include CPACK for creating DEB and RPM packages
+set(CPACK_PACKAGE_NAME "notes")
+set(CPACK_PACKAGE_VERSION ${PROJECT_VERSION}${GIT_REV})
+set(CPACK_PACKAGE_FILE_NAME ${CPACK_PACKAGE_NAME}_${CPACK_PACKAGE_VERSION}_${CPACK_DEBIAN_PACKAGE_ARCHITECTURE})
+set(CPACK_PACKAGE_CONTACT "Nutty Artist <awesomeness.notes@gmail.com>")
```

@nuttyartist Can we change this to:

```suggestion
set(CPACK_PACKAGE_CONTACT "Ruby Mamistvalove <ruby.mamistvalove@gmail.com>")
```

?